Ve/Sniffs/ControlStructures/ControlSignatureSniff.php

```diff
@@ -81,12 +88,12 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
 	}
 
 	/**
-	 * @param array $tokens
+	 * @param array   $tokens
 	 * @param integer $stackPtr
-	 * @param PHP_CodeSniffer_File $phpcsFile
+	 * @param File    $phpcsFile
 	 */
 	private function checkSingleSpaceAfterKeyword(array $tokens, $stackPtr,
-											   PHP_CodeSniffer_File $phpcsFile)
+											   File $phpcsFile)
```

Comment:
Are there 2 lines or is it just github looking odd?

Reply:
Yeah it was split over two lines, probably because the line was longer before. The sad thing is, this repo doesn't obey any coding standards itself at all... 😞

Reply:
![image](https://user-images.githubusercontent.com/2286713/40660162-1a5bd498-6348-11e8-96bb-aa4b19e46285.png)